docs-ui/components/sample.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+
+const Sample = styled('div')`
+  border-radius: ${p => p.theme.borderRadius};
+  border: dashed 1px ${p => p.theme.gray200};
+  padding: ${space(1)} ${space(2)};
+  margin: ${space(2)} 0;
+
+  & > *:first-of-type {
+    margin-top: 0;
+  }
+
+  & > *:last-of-type {
+    margin-bottom: 0;
+  }
```

Comment:
Why not combine these selectors?

```css
  & > *:first-of-type,
  & > *:last-of-type {
    margin-top: 0;
  }
```

Reply:
Oh that's cause the `first-of-type` eliminates `margin-top`, while `last-of-type` is for `margin-bottom`.